### DISCUSSION TITLE
validation-in-aspnet-mvc-part-4-example-with-server-side/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#25] [Blog comments]
# validation-in-aspnet-mvc-part-4-example-with-server-side/

https://blogs.taiga.nl/martijn/posts/validation-in-aspnet-mvc-part-4-example-with-server-side/

**Comment by disqus-comments-migrator** [comment]
<em>cypher commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at januari 02 2009, 07:08</em>

---

<p>Thanks for submitting this sample. <br>Very helpful. <br><br>Cheers. <br></p>

**Comment by disqus-comments-migrator** [comment]
<em>cypher commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at januari 02 2009, 07:57</em>

---

<p>I don't clearly understand it all, but as far as I know, you declare/add "options" (in AddParameterToOptions()), but never output them as Javascript (in CreateBeforeFormClosed()). <br><br>My two cents.</p>

**Comment by disqus-comments-migrator** [comment]
<em>martijn commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at januari 02 2009, 08:07</em>

---

<p>Those options were just copied from the Monorail JQuery validator and as far as I can see, you could remove these.</p>
